### PR TITLE
Feature/plop

### PR DESCRIPTION
**Pull request by grmillsy1**
Added plop to generate component templates.

### CODE REVIEW

File: PLOP.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#Plop
+
+Plop is a small component generator that can be used to quickly set up our web-components from pre designed templates.
+
+### Usage
+##### Component
+To use plop to generate a component follow these steps:
+
+````
+1. $ npm run generate component
```
Comment (jennasalau) [inline PLOP.md]: We are favouring "yarn" in this project

File: PLOP.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+2. Choose a component name. i.e Button (please leave out Rui or any prefix at this stage)
+3. Chose if you would like to add an adapter. If you choose no, this can be done at a later stage.
+````
+Hey presto! Your component can now be fount in the components directory.
+
+##### Adapter
+The adapters are React and Vue.js wrappers for our web components. To create an adaptor for your component follow these steps.
+
+````
+  1. $ npm run generate adapter
```
Comment (jennasalau) [inline PLOP.md]: Yarn here too please

File: PLOP.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  2. Select which adapter you require. (React, Vue or both)
+  3. Enter name of component you wish to create an adaptor for (leaving off any Rui prefix).
+````
+*Please note: Creating an adapter will only work for an existing component.
```
Comment (jennasalau) [inline PLOP.md]: Can you please write a script that lists all the folders that don't have "Vue" or "React" affixed so you can just select the right one.

I'll send you an example

File: plopfile.js [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                actions = actions.concat(vueActions)
+            }
+            return actions
+        },
+    })
+};
+
+
+
+
```
Comment (jennasalau) [inline plopfile.js]: This is super neat!! Nice one.

File: PLOP.md (hunk 1 above)
Comment (grmillsy1) [reply PLOP.md]: Changed this to 'yarn generate component'

File: PLOP.md (hunk 2 above)
Comment (grmillsy1) [reply PLOP.md]: Changed this to 'yarn generate adapter'

File: PLOP.md (hunk 3 above)
Comment (grmillsy1) [reply PLOP.md]: @jennasalau  have changed this to find all the components without an adapter, so far have just done it for react but vue can be added in easily when we need it.

File: lerna.json [hunk 5]
```diff
@@ -1,9 +1,11 @@
 {
   "lerna": "2.7.1",
   "packages": [
-    "components/**",
-    "tools/**",
-    "www"
+	  "components/**",
+	  "reactComponents/**",
```
Comment (jennasalau) [inline lerna.json]: Can we rename them to just "vue" and "react" pretty please. Just incase we put utils or other things in there. Other than that im happy with this PR

Comment (grmillsy1) [reply lerna.json]: Sure, have changed these

**Pull request merged**